test/parallel/test-http2-util-update-options-buffer.js

```diff
@@ -25,7 +25,8 @@ const IDX_OPTIONS_MAX_SESSION_MEMORY = 8;
 const IDX_OPTIONS_MAX_SETTINGS = 9;
 const IDX_OPTIONS_STREAM_RESET_RATE = 10;
 const IDX_OPTIONS_STREAM_RESET_BURST = 11;
-const IDX_OPTIONS_FLAGS = 12;
+const IDX_OPTIONS_STRICT_HTTP_HEADER_FIELD_VALIDATION = 12;
```

Comment:
Nit: this still has the old option name (just test code, but nice to be consistent)